app/views/teachers/tasks/_form.html.slim

```diff
@@ -0,0 +1,4 @@
+= simple_form_for [:teachers, :course, :lesson, task] do |f|
```

Comment:
@latypova-alina 
in `Teachers::TasksController#new` you create `@task` variable, but here you use `task` var
does it work right?